charts/oidc-proxy/ci/testing-values.yaml

```diff
@@ -5,6 +5,7 @@ env:
   PROXY_URL3: http://test-datarepo-api.integration-temp:8080/ga4gh
   PROXY_PATH3: /ga4gh
   LOG_LEVEL: debug
+  CORS_REGEX: http(s)?://(www\\.)?(.+\\.broadinstitute\\.org|.+\\envs-terra\\.bio|.+\\.terra\\.bio|.+\\.appspot\\.com)$
```

Comment:
Just thought of one more question - do we need to include `data.humancellatlas.org` here? (I'm not sure how HCA teams are interacting with the Data Repo)

Reply:
Yes! I reached out to HCA and that was their recommendation, so I added it too.

Reply:
Awesome!